practical/DeepClustering/deep_clustering_dummy.py

```diff
@@ -1,2 +1,23 @@
+import torch
+
+
 def hello_world(string: str) -> str:
     return "Hello " + string
+
+
```

Comment:
maybe it would be possible to stop the iteration earlier when the centers do not change anymore and to do some input vaidation of the given parameters. But I like you short implementation which is much more clearer than mine!